app/src/main/java/com/zulip/android/database/DatabaseHelper.java

```diff
@@ -50,6 +51,7 @@ public void onCreate(SQLiteDatabase db, ConnectionSource connectionSource) {
             TableUtils.createTable(connectionSource, Stream.class);
             TableUtils.createTable(connectionSource, Message.class);
             TableUtils.createTable(connectionSource, MessageRange.class);
+            TableUtils.createTable(connectionSource, Emoji.class);
```

Comment:
If you bump the DATABASE_VERSION number, this will correctly rebuild your database for upgrades. :)